### PR TITLE
Fixed slash as a division SASS [Fixed in FA6 - our fix does not break libsass - Will be fixed in FA5 (No ETA, sorry)]

### PR DESCRIPTION
**Pull request by Okipa**
This PR fixes the slash-as-a-division SASS coming breaking change.

To re-execute these changes from your side, you just have to follow these simple recommandations : https://sass-lang.com/documentation/breaking-changes/slash-div

Thanks to you for applying this !

### CODE REVIEW

File: js-packages/@fortawesome/fontawesome-free/scss/_larger.scss [hunk 1]
```diff
@@ -2,9 +2,11 @@
 // -------------------------
 
 // makes the font 33% larger relative to the icon container
+@use "sass:math";
+
 .#{$fa-css-prefix}-lg {
-  font-size: (4em / 3);
-  line-height: (3em / 4);
+  font-size: math.div(4em, 3);
+  line-height: (3em * 0.25);
```
Comment (MisatoTremor) [inline js-packages/@fortawesome/fontawesome-free/scss/_larger.scss:9]: This should be written without a leading zero `line-height: (3em * .25);`

File: js-packages/@fortawesome/fontawesome-free/scss/_list.scss [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 .#{$fa-css-prefix}-ul {
   list-style-type: none;
-  margin-left: $fa-li-width * 5/4;
+  margin-left: $fa-li-width * 5*0.25;
```
Comment (MisatoTremor) [inline js-packages/@fortawesome/fontawesome-free/scss/_list.scss:6]: Can be simplyfied to `margin-left: $fa-li-width * 1.25;`

File: scss/_larger.scss [hunk 3]
```diff
@@ -2,9 +2,11 @@
 // -------------------------
 
 // makes the font 33% larger relative to the icon container
+@use "sass:math";
+
 .#{$fa-css-prefix}-lg {
-  font-size: (4em / 3);
-  line-height: (3em / 4);
+  font-size: math.div(4em, 3);
+  line-height: (3em * 0.25);
```
Comment (MisatoTremor) [inline scss/_larger.scss:9]: This should be written without a leading zero `line-height: (3em * .25);`

File: scss/_list.scss [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
 .#{$fa-css-prefix}-ul {
   list-style-type: none;
-  margin-left: $fa-li-width * 5/4;
+  margin-left: $fa-li-width * 5*0.25;
```
Comment (MisatoTremor) [inline scss/_list.scss:6]: Can be simplyfied to `margin-left: $fa-li-width * 1.25;`

**Review by MisatoTremor — CHANGES_REQUESTED** [review]
Great change! Just some small remarks.

File: js-packages/@fortawesome/fontawesome-free/scss/_larger.scss (hunk 1 above)
Comment (Okipa) [reply js-packages/@fortawesome/fontawesome-free/scss/_larger.scss]: @MisatoTremor Feedback applied 👍

File: js-packages/@fortawesome/fontawesome-free/scss/_list.scss (hunk 2 above)
Comment (Okipa) [reply js-packages/@fortawesome/fontawesome-free/scss/_list.scss]: @MisatoTremor Feedback applied 👍

File: scss/_larger.scss (hunk 3 above)
Comment (Okipa) [reply scss/_larger.scss]: @MisatoTremor Feedback applied 👍

File: scss/_list.scss (hunk 4 above)
Comment (Okipa) [reply scss/_list.scss]: @MisatoTremor Feedback applied 👍

**Comment by Okipa** [comment]
By the way, if this is merged, this is not too late for a little `hacktoberfest-accepted` tag ;)

Thanks and keep up the good work !

**Comment by C-Lodder** [comment]
You're limiting **everyone** to dart-sass by using `@use "sass:math";` and `math.div()`, where as millions of people are still using libsass

**Comment by tagliala** [comment]
> You're limiting **everyone** to dart-sass by using `@use "sass:math";` and `math.div()`, where as millions of people are still using libsass

Hi @C-Lodder, the implemented fix uses a custom divide function which does not break libsass compatibility

FA6 beta 2 has been fixed, FA5 is still in the todo-list

I'm going to close this PR

**Comment by dleavitt** [comment]
why was this closed? is the new approach incompatible with older libsass versions?

**Comment by tagliala** [comment]
> why was this closed?

This has been fixed upstream on FA6 and will be backported on FA5

> is the new approach incompatible with older libsass versions?

Nope, the new approach will be compatible

**Comment by tagliala** [comment]
reopening for tracking purposes

**Comment by rickerd** [comment]
Merge this! It's super annoying when compiling. All the warnings that can easily be fixed merging this.

**Comment by rickerd** [comment]
When will FA6 B3 come out? It's sooo annoying.
+ thanks for the speedy reply.

**Comment by tagliala** [comment]
> When will FA6 B3 come out? It's sooo annoying.

The third beta of FA6 has been already released: https://fontawesome.com/v6.0

**Comment by rickerd** [comment]
I see, but there's no repo/tag/release for version 6.
Or am I missing things?

**Comment by tagliala** [comment]
> I see, but there's no repo/tag/release for version 6.

Correct, npm packages and downloadable assets are available but the github repo does not have the code yet

**Comment by tagliala** [comment]
This has been fixed in FA6 and the fix will be backported to FA5

No ETA, sorry

Locking here

**Pull request closed without merging**